pkgs/applications/misc/remarkable/reStream/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, bash
+, stdenv
+, lz4
+, ffmpeg-full
+, fetchFromGitHub
+, openssh
+}:
+
+stdenv.mkDerivation rec {
+  pname = "reStream";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "rien";
+    repo = pname;
+    rev = version;
+    sha256 = "18z17chl7r5dg12xmr3f9gbgv97nslm8nijigd03iysaj6dhymp3";
+  };
+
+  buildInputs = [ ffmpeg-full lz4 bash openssh ];
```

Comment:
This doesn't do anything for a shell script, except for `bash` which I guess is patched in as the shebang. You should use `makeWrapper`.

Reply:
Roger that. Thanks.